CleanWeather/Helpers/NoActionsTextField.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  NoActionsTextField.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 03/12/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+class NoActionsTextField: UITextField {
```

Comment:
You've named this textfield to 'NoActions...' but you're giving an option to turn them on.
How about just a 'SharedTextField' or 'MainTextField' with option for future changes?